src/components/MyItemsPage.jsx

```diff
@@ -3,13 +3,15 @@ import React from 'react'
 import AddItemForm from './AddItemForm'
 import ItemContainer from './ItemContainer'
 import NavBar from './NavBar'
+import MyItemsNavBar from './MyItemsNavBar'
 
 import '../styles.css'
 
 function MyItemsPage () {
   return (
     <>
       <NavBar />
+      <MyItemsNavBar />
```

Comment:
React bootstrap has a Tabs component, might be simpler to use that instead of having a second navbar.

https://react-bootstrap.github.io/components/tabs/

Reply:
I'm going to keep it as a navbar for now, I can reuse that navbar between the two pages. Gave tabs a try and it broke the app - but we could do a refactor later and try it out?